### PR TITLE
Pass env variable to cf app

### LINKED ISSUE #703: CF Manifest doesn't set env
We simply forget to read a pass on env variables from a template/domain class.

### PR DESCRIPTION
**Pull request by jvalkeal**
- Simple pass on the map from template and domain
  structure into app builder.
- Fixes #703

### CODE REVIEW

File: spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/cloudfoundry/CloudFoundryManifestApplicationDeployer.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public ApplicationManifest getCFApplicationManifest(Release release) {
 					.disk(CloudFoundryApplicationManifestUtils.memoryInteger(manifest.getDiskQuota()))
 					.domains(manifest.getDomains() != null && !manifest.getDomains().isEmpty() ? manifest.getDomains()
 							: null)
-					.environmentVariables(new HashMap<>())
+					.environmentVariables(manifest.getEnv() != null ? manifest.getEnv() : null)
```
Comment (ilayaperumalg) [inline spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/cloudfoundry/CloudFoundryManifestApplicationDeployer.java]: @jvalkeal Have you tried passing the env via `properties` cmdline option?

Comment (jvalkeal) [reply spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/cloudfoundry/CloudFoundryManifestApplicationDeployer.java]: Yes `package install --package-name logcf --release-name mylogcf --platform-name cf-dev --package-version 1.0.0 --properties "spec.manifest.env={key1: value1, key2: value2}"` worked just fine.

**Comment by ilayaperumalg** [comment]
LGTM. There is an import issue which caused the travis CI build failure. Will fix and merge.

**Comment by ilayaperumalg** [comment]
Rebased and merged as af557e5

**Pull request closed without merging**